packages/cli/src/utils/deploy-v2.ts

```diff
@@ -102,6 +108,7 @@ export async function deploy(mudConfig: MUDConfig, deployConfig: DeployConfig):
   console.log(chalk.blue("Installing core World modules"));
   await fastTxExecute(WorldContract, "installRootModule", [await modulePromises.CoreModule, "0x"]);
   await fastTxExecute(WorldContract, "installRootModule", [await modulePromises.RegistrationModule, "0x"]);
+  await fastTxExecute(WorldContract, "installModule", [await modulePromises.UniqueEntityModule, "0x"]);
```

Comment:
I think while this is a very common usecase, we should still allow users to decide to include or not include it, as it's not strictly necessary for the World contract to function (unlike `CoreModule` and `RegistrationModule`). If we remove this line, consumers have to add the module to their `mud.config.mts`, which is slightly inconvenient, but can be made more convenient with proper config plugin support in the near future.